sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -4946,6 +4946,17 @@ pi_result piEnqueueMemBufferWriteRect(
       EventWaitList, Event);
 }
 
+// This is an experimental option to test performance of device to device copy
+// operations on copy engines (versus compute engine)
+static bool UseCopyEngineForD2DCopy(bool HasCopyEngine) {
```

Comment:
Please follow `ZeSerialize` in how to init/query env var.

Reply:
Thanks Sergey. I have modified the code accordingly.